R-package/tests/helper_scripts/install_deps.R

```diff
@@ -3,7 +3,9 @@
 ## inconsistent is found.
 pkgs <- c(
   ## CI
-  "devtools",
+  "caret",
+  "pkgbuild",
```

Comment:
added `{pkgbuild}` (one of the other packages wrapped by `{devtools}`) to fix this:

> Run python3 tests/ci_build/test_r_package.py --r=/tmp/R-devel/bin/R --task=doc
ℹ Loading xgboost
Error in pkgload::load_all(path, helpers = FALSE, attach_testthat = FALSE) : 
  The package `pkgbuild` is required to compile packages with a `src/`
directory.

([build link to failing CI job](https://github.com/dmlc/xgboost/actions/runs/3635329755/jobs/6134326110))